deployments/nomad/README.md

```diff
@@ -0,0 +1,72 @@
+# Splunk OpenTelemetry Collector on HashiCorp Nomad
```

Comment:
```suggestion
# Splunk OpenTelemetry Collector for HashiCorp Nomad
```

Reply:
I'm not sure I agree with this since we aren't providing a fully capable installation path or integration w/ nomad.